### DISCUSSION TITLE
helloworld example does not loading in Extension Development Host

### DISCUSSION THREAD

**Discussion by MrThanlon** [microsoft/vscode-discussions#760] [Extension Development QnA] [answered]
Same as https://stackoverflow.com/questions/75759492/vs-code-extension-not-loading-in-extension-development-host, but upgrading vscode doesn't work.

I generated a sample extension by following https://code.visualstudio.com/api/get-started/your-first-extension, and press F5 to run, but the new vscode window doesn't load helloworld extensions, and there is no `helloworld` command.

Version: 1.80.2 (user setup)
Commit: 2ccd690cbff1569e4a83d7c43d45101f817401dc
Date: 2023-07-27T20:40:28.909Z
Electron: 22.3.14
ElectronBuildId: 22695494
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Windows_NT x64 10.0.19045

**Comment by MrThanlon** [comment, marked as answer]
fixed, package.json defined the version requirement of extenstion, change it to fit current version.

**Reply by zty012** [reply]
fixed my issue, thanks!!
